src/login/meson.build

```diff
@@ -96,12 +96,19 @@ modules += [
                 'name' : 'pam_systemd',
                 'conditions' : ['HAVE_PAM'],
                 'sources' : files('pam_systemd.c'),
+                'dependencies' : [
+                        libpam_misc,
+                        libpam,
+                ],
                 'version-script' : meson.current_source_dir() / 'pam_systemd.sym',
         },
         pam_template + {
                 'name' : 'pam_systemd_loadkey',
                 'conditions' : ['HAVE_PAM'],
                 'sources' : files('pam_systemd_loadkey.c'),
+                'dependencies' : [
+                        libpam,
+                ],
```

Comment:
Also here. This is not necessary.

Reply:
well, it's a pam module, it's really weird, and pointless to avoid linking against libpam, because libpam after all loads us. it's always going to be there, hence I figured it makes sense to make the dep explicit.

(or to say this differently: we could choose to not bother with the sym_xyz for the libpam symbols we call from pam_systemd.c, but that would mean, using our various helpers that we want to use from both our pam modules and our pam services would have to be duplicated. hence i opted to always go the dlopen() route, but also declare the dep explicitly. will add comment)